src/Mbstring/bootstrap.php

```diff
@@ -133,27 +133,27 @@ function mb_str_split($string, $length = 1, $encoding = null) { return p\Mbstrin
 }
 
 if (!function_exists('mb_str_pad')) {
-    function mb_str_pad(string $string, int $length, string $pad_string = ' ', int $pad_type = STR_PAD_RIGHT, ?string $encoding = null): string { return p\Mbstring::mb_str_pad($string, $length, $pad_string, $pad_type, $encoding); }
```

Comment:
what about a bootstrap80.php file to add the return types when running on PHP8+?

Reply:
That file already exists. ✌🏻

Reply:
ah indeed :) 
there are some failure to fix ;)

Reply:
Can you help we with those? Apparently, the test listener is skipping tests at a strange point in time which conflicts with the `expectWarning()` feature of PHPUnit. How do we solve that usually?